cmake/SetupBlaze.cmake

```diff
@@ -57,6 +57,28 @@ if(SLEEF_FOUND)
   set(_BLAZE_USE_SLEEF 1)
 endif()
 
+# If BLAZE_USE_STRONG_INLINE=ON, Blaze will use this keyword to increase the
```

Comment:
Sorry I didn't think of this before, but would you mind documenting these CMake flags in `docs/DevGuide/BuildSystem.md`

Reply:
Do you have a preference where it is added? I wouldn't think it fits under "Commonly Used CMake flags" but not sure if it deserves its own section like "Inlining Configuration" (I plan to submit another PR to toggling `SPECTRE_ALWAYS_INLINE`) or "Blaze Configuration" or "Additional CMake flags" (which alphabetically unfortunately would appear before the "Commonly used CMake flags" section)

Reply:
I would put it in `Commonly used..` We'll have to revisit that list at some point, but for now I think there in alphabetical order is fine